### PR TITLE
Updated comments for TaintBasedEvictions.

### PR DESCRIPTION
**Pull request by gyliu513**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

### CODE REVIEW

File: staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/well_known_labels.go [hunk 1]
```diff
@@ -34,12 +34,12 @@ const (
 	// fluentd in not running as a manifest pod with LabelFluentdDsReady.
 	LabelFluentdDsReady = "alpha.kubernetes.io/fluentd-ds-ready"
 
-	// When the --use-taint-based-evictions flag is enabled,
+	// When the --feature-gates=TaintBasedEvictions=true flag is enabled,
```
Comment (gmarek) [inline staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/well_known_labels.go]: This probably should be worded differently, as --feature-gates=FooBar=true,TaintBasedEviction=true is also good.

I guess something along the lines:
"When feature-gate for TaintBasedEvictions=true..."

Comment (gyliu513) [inline staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/well_known_labels.go]: Updated to `When feature-gate for TaintBasedEvictions=true flag is enabled,`, hope it is OK. Thanks @gmarek

**Comment by gmarek** [comment]
This is a cleanup PR, which is 100% safe and should go in in 1.6.

**Comment by gyliu513** [comment]
@k8s-bot cvm gce e2e test this

**Comment by gyliu513** [comment]
@k8s-bot cvm gce e2e test this

**Pull request merged**